test-requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
+pytest-cov
+codecov
```

Comment:
I don't have a good handle on conventions around `test-requirements.txt` -- is it normal to have stuff in here that's only needed for CI (codecov), not for running the tests locally?